src/app/Main.js

```diff
@@ -11,9 +11,18 @@ import { serverApiRecordEvent } from 'app/utils/ServerApiClient';
 import * as steem from '@steemit/steem-js';
 import { determineViewMode } from 'app/utils/Links';
 
-window.onerror = error => {
-    if (window.$STM_csrf) serverApiRecordEvent('client_error', error);
-};
+window.addEventListener('error', error => {
```

Comment:
Can we name this argument `event`  or `errorEvent` per:
https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent

Reply:
good idea; will do